.github/workflows/auto-assigning.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto Author Assign
+
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
```

Comment:
No need to add a reopening trigger due to the author already being added when the Pull Request is initially created.